functests/ptp/ptp.go

```diff
@@ -39,71 +40,81 @@ const (
 	ptpDaemonsetName                = "linuxptp-daemon"
 )
 
+var discoveryMode bool
+
+func init() {
```

Comment:
Can this be in it's own package?
Something like `discoverymode.Enabled()`
This would make it easier to port it on the other test suites.

Reply:
Added to the execute package (as it's connected to test execution).
Let me know if you prefer a completely new package for the oneliner function.

Reply:
Fine be me. 
As a general rule, there's no harm in having small packages and it improves readability.